test-cases/RMLTC0010c-CSV/mapping.ttl

```diff
@@ -19,5 +19,5 @@
 
   rr:predicateObjectMap [
     rr:predicate ex:code ;
-    rr:objectMap [ rr:template "\{\{\{ {ISO 3166} \}\}\}"; rr:termType rr:Literal]
+    rr:objectMap [ rr:template "\\{\\{\\{ {\"ISO 3166\"} \\}\\}\\}"; rr:termType rr:Literal; ]
```

Comment:
Why are double quotes added?

Reply:
To make it in line with the R2RML test case https://www.w3.org/2001/sw/rdb2rdf/test-cases/#R2RMLTC0010c I assume

Reply:
If that would be the case, then it should only happen for SQL, because for CSV/JSON/XML it will actually look for `"ISO 3166"`, which will fail.

Reply:
Guess it all depends on the referenceFormution. I agree on SQL, for XML, see my other comment, for CSV I guess it depends on how this is defined, but I guess we can assume it should indeed be without quotes (which is [in line with CSVW](https://www.w3.org/TR/tabular-data-primer/#documentation-columns)), for JSON, I guess we need to make clear that the reference will always be quoted to avoid confusion in [the RML spec](https://rml.io/specs/rml/#example-JSON)?